### PR TITLE
fix bug where wrong shell was being passed to windows

### PR DESCRIPTION
**Pull request by rosskirkpat**

### CODE REVIEW

File: commands/create.go [hunk 1]
```diff
@@ -539,11 +539,11 @@ func updateUserdataFile(driverOpts *rpcdriver.RPCFlags, machineName, userdataFla
 // writeCloudConfig sets custom install script path to the runcmd directive
 // and passes the script path to commonCloudConfig
 func writeCloudConfig(machineName, encodedData, machineOS string, cf map[interface{}]interface{}, newUserDataFile *os.File) error {
-	path := "/usr/local/custom_script/install.sh"
+	path := "sh /usr/local/custom_script/install.sh"
```
Comment (thedadams) [inline commands/create.go:542]: nit: path is no longer a good name for this variable.

**Review by thedadams — APPROVED** [review]

**Review by phillipsj — APPROVED** [review]
LGTM!

**Review by phillipsj — APPROVED** [review]
LGTM!

File: commands/create.go (hunk 1 above)
Comment (rosskirkpat) [reply commands/create.go]: I switched to using `command`

**Review by thedadams — APPROVED** [review]

**Pull request merged**